lib/pliny/middleware/versioning.rb

```diff
@@ -44,13 +44,14 @@ def detect_api_version(env)
       env['HTTP_ACCEPT'] = media_types.join(', ')
 
       version ||= @default
-      set_api_version(env, version)
+      set_api_version(env, version, variant)
       @app.call(env)
     end
 
-    def set_api_version(env, version)
+    def set_api_version(env, version, variant)
       # API modules will look for the version in env
       env["HTTP_X_API_VERSION"] = version
```

Comment:
I'm thinking we should just do it. The only way that header was officially exposed is through the `version` method in the router.